.github/workflows/ci.yml

```
@@ -215,6 +201,43 @@ jobs:
+          --tag main \
+          --tag production \
+          --branch ${{ github.ref_name }}
+      env:
+        PACT_BROKER_URL: ${{ secrets.PACT_BROKER_URL }}
+        PACT_BROKER_TOKEN: ${{ secrets.PACT_BROKER_TOKEN }}
+        GIT_COMMIT: ${{ github.sha }}
+        GIT_BRANCH: ${{ github.ref_name }}
+
+    - name: Publish Pacts to Broker (PR/Develop)
+      if: matrix.test-type == 'contracts' && (github.ref == 'refs/heads/develop' || github.event_name == 'pull_request')
+      run: |
```

actionlint (1.7.8)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)